docs/section5/Rule5-37.md

```diff
@@ -0,0 +1,64 @@
+
+# Envelope - Rule 5-37  
+
+**Rule ID:** 5-37  
+**Rule Description:** Skylight U-factors for residential, non-residential and semi-heated spaces in the baseline model must match the appropriate requirements in Table G3.4-1 through G3.4-8.  
+**Rule Assertion:** B-RMR subsurface: U_factor = expected value  
+**Appendix G Section:** Section G3.1-5(e) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** Tables G3.4-1 to G3.4-8  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Tables G3.4-1 to G3.4-8  
+**Function Call:**
+
+  1. get_surface_conditioning_category()  
+  2. get_opaque_surface_type()  
+  3. get_building_skylight_roof_areas()
+
+## Rule Logic:  
+
+- Get building climate zone: `climate_zone = B_RMR.weather.climate_zone`  
+
+- Get surface conditioning category dictionary for B_RMR: `scc_dictionary_b = get_surface_conditioning_category(B_RMR)`  
+
+- Get building skylight roof areas dictionary: `skylight_roof_areas_dictionary_b = get_building_skylight_roof_areas(B_RMR)`
+
+- For each building segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`  
+
+  - Calculate building segment skylight roof ratio: `srr_b = skylight_roof_areas_dictionary_b[building_segment_b.id][0] / skylight_roof_areas_dictionary_b[building_segment_b.id][1]`
+
+    - If skylight roof area is 2% or less, set skylight type to "0%-2.0%": `if srr_b <=0.02: skylight_type_b = "0%-2.0%"`
+
+    - Else, set skylight type to "2.1%+": `else: skylight_type_b = "2.1+"`
+
+  - For each thermal_block in building segment: `for thermal_block_b in building_segment_b.thermal_blocks:`  
+
+    - For each zone in thermal block: `for zone_b in thermal_block_b.zones:`  
+
+      - For each surface in zone: `for surface_b in zone_b.surfaces:`  
+
+        - Check if surface is roof or ceiling and is regulated, get surface conditioning category: `if ( get_opaque_surface_type(surface_b) == "ROOF" ) AND ( scc_dictionary_b[surface_b] != "UNREGULATED" ): scc_b = scc_dictionary_b[surface_b]`
+
+          - If surface is exterior residential, exterior non-residential, or semi-exterior, get baseline skylight construction U-factor from Table G3.4-1 to G3.4-8 based on climate zone, surface conditioning category and skylight type: `if scc_b in ["EXTERIOR RESIDENTIAL", "EXTERIOR NON-RESIDENTIAL", "SEMI-EXTERIOR"]: target_u_factor = data_lookup(table_G3_4, climate_zone, scc_b, "SKYLIGHT", skylight_type_b, "ASSEMBLY MAX. U")`  
+
```

Comment:
I would replace "... and skylight type" with " ... and skylight percentage of roof area".

Reply:
Updated.